frontend/www/js/omegaup/common/navbar_v2.ts

```diff
@@ -7,6 +7,20 @@ import Vue from 'vue';
 
 OmegaUp.on('ready', () => {
   const payload = types.payloadParsers.CommonPayload('header-payload');
+  const localMetaData = JSON.parse(
+    localStorage.getItem('toggle-bootstrap-theme') ?? '',
```

Comment:
```suggestion
    localStorage.getItem('toggle-bootstrap-theme') ?? '{}',
```

si no, `JSON.parse('')` arroja una excepción.

Reply:
Listo